Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -400,9 +401,14 @@ void APlayCppSdkActor::ClearSession(bool &success) {
     IFileManager &FileManager = IFileManager::Get();
     success =
         FileManager.Delete(*(FPaths::ProjectSavedDir() + "sessioninfo.json"));
-    _coreClient = NULL;
+    destroyCoreClient();
```

Comment:
destroyCoreClient is called from DestroyClient?

Reply:
This is called in `ClearSession` where will clear all sessions, including QR code request and all other related network requests. Without it, the pending network requests can not be cleared after calling `ClearSession`.

Reply:
aha, i see

Reply:
destroyCoreClient can not destroy pending network requests...